### ISSUE TITLE
First launch, terminal panel looks empty (but it's not)

### ISSUE THREAD

**Issue by albinotonnina** [Duroktar/vscode-npm-scripts#7]
Hi, thank you very much for this plugin, I love it!

I noticed this, lately:
When I launch a command for the first time (ie, there is no terminal panel for the launched command yet) the tab opens but looks empty:
![image](https://user-images.githubusercontent.com/37283/39424935-e1cbb262-4c70-11e8-996a-e552a4817fbd.png)

opening another tab, ie: Debug Console and re-opening the Terminal tab solves the issue.
I'm on a mac, npm-scripts default configuration.

Cheers!

**Comment by frankMazzarella** [comment]
I can confirm this has been an issue for me as well using multiple machines running Windows and Linux

**Comment by frankMazzarella** [comment]
Follow-up: this is intermittent and not related to the latest release.

**Comment by Duroktar** [comment]
I'm really sorry for not responding sooner. I've been so busy lately and this must have slipped past my radar. I can confirm that this has been an issue for me as well, mostly on a Linux build but it happens on my mac as well. I have a few other things I need to fix up when I can get around to it and I'll be sure to check into this as well. There has been a lot of movement on the vscode front so it wouldn't surprise me at all if I'm using a deprecated API somewhere.

I think it's awesome that you love the plugin and I hope my late response hasn't tarnished that at all. I'll be sure to ping this issue when I start making fixes.

Cheers! 🍻

**Comment by albinotonnina** [comment]
No problem at all!
Your plugin is great but did you know that vscode 1.23.0 introduced a similar panel?

**Comment by Duroktar** [comment]
Yes I did! I actually threw a mention on the main README. After trying it though, I think mine still has its uses. I found that with the built in functionality having to right click the script from the sidebar not intuitive at all since with my extension it's just click and go. 

_BUT_, I hate to say it, but this extension regressed a bit when I added support for workplaces :( I got sloppy and didn't re-implement the file change watcher for added/removed scripts. So that's a reload (pain in the ass). Plus it losses control of the console after closing the open instance.. It didn't do that before either (I think it was #1 that fixed that actually, so sorry to the author of that one).

I'm rambling lol Thanks again for the support kind sir :) I hope to see you around :beers:
